### DISCUSSION TITLE
branding: tagline and other terminology

### DISCUSSION THREAD

**Discussion by lostmygithubaccount** [ibis-project/ibis#7065] [General]
With the merge of the Quarto docs, we now have the tagline "the portable Python dataframe library". 

I like it, but want to open a discussion for further input openly. I think "portable" is good as it is the key differentiating feature of Ibis and works well with the modular and composable data ecosystem we're looking to exist within.

If we settle on this or another, should update the GitHub About:

<img width="330" alt="image" src="https://github.com/ibis-project/ibis/assets/54814569/2ade50d2-1af3-43a7-b5a7-4ddde17ec065">

We don't necessarily have to do anything formal but might want to discuss related terminology we should use to have some amount of consistency when talking about Ibis. Feel free to provide opinions here!

**Comment by NickCrews** [comment]
Oops, I opened https://github.com/ibis-project/ibis/discussions/7137 (which now is more active) before I saw this. But I agree, the "GitHub about" should get updated when we choose.

**Reply by lostmygithubaccount** [reply]
no worries let's just close this one
